app/src/main/res/values-hu/strings.xml

```diff
@@ -129,56 +129,56 @@
+    <string name="untilEnd_bluehour">kék óra végéig</string>
+    <string name="until_moonrise">holdkeltéig</string>
+    <string name="until_moonset">holdnyugtáig</string>
+    <string name="until_moonnoon">holdi délig</string>
+    <string name="until_moonnight">holdi éjfélig</string>
 
     <!-- Table -->
     <string name="sunrise">Napkelte</string>                             <!-- contentDescription -->
     <string name="sunrise_short">Napkelte</string>                          <!-- table title -->
     <string name="sunset">Napnyugta</string>                               <!-- contentDescription -->
     <string name="sunset_short">Napnyugta</string>                            <!-- table title -->
-    <string name="moonrise">Moonrise</string>                             <!-- contentDescription -->  <!-- TODO -->
-    <string name="moonrise_short">Moonrise</string>                             <!-- table title -->  <!-- TODO -->
-    <string name="moonset">Moonset</string>                             <!-- contentDescription -->  <!-- TODO -->
-    <string name="moonset_short">Moonset</string>                             <!-- table title -->  <!-- TODO -->
-
+    <string name="moonrise">Holdkelte</string>                             <!-- contentDescription -->
+    <string name="moonrise_short">Holdkelte</string>                             <!-- table title -->
+    <string name="moonset">Holdnyugta</string>                             <!-- contentDescription -->
+    <string name="moonset_short">Holdnyugta</string>                             <!-- table title -->
+    
     <!-- Time spans: as seen in length of day, today/tomorrow time delta, and note. -->
     <string name="delta_format"><xliff:g id="value" example="3">%1$s</xliff:g><xliff:g id="units" example="m">%2$s</xliff:g></string>
     <string name="delta_years">év</string>    <!-- e.g. 1y -->
-    <string name="delta_weeks">w</string>    <!-- e.g. 2w -->  <!-- TODO -->
+    <string name="delta_weeks">hét</string>    <!-- e.g. 2w -->
     <string name="delta_days">nap</string>     <!-- e.g. 1d -->
     <string name="delta_hours">óra</string>    <!-- e.g. 2h -->
     <string name="delta_minutes">perc</string>  <!-- e.g. 3m -->
     <string name="delta_seconds">mp</string>  <!-- e.g. 4s -->
 
-    <string name="hence"><xliff:g id="timeSpan">%1$s</xliff:g>  from now</string>  <!-- e.g. 15d 23h from now -->  <!-- TODO -->
-    <string name="ago"><xliff:g id="timeSpan">%1$s</xliff:g>  ago</string>         <!-- e.g. 1y 15d 12h ago -->  <!-- TODO -->
+    <string name="hence"><xliff:g id="timeSpan">%1$s</xliff:g>  múlva</string>  <!-- e.g. 15d 23h from now -->
+    <string name="ago"><xliff:g id="timeSpan">%1$s</xliff:g>  óta</string>         <!-- e.g. 1y 15d 12h ago -->
 
     <!-- misc time values: might be displayed in a timeField in place of an actual time -->
     <string name="time_none">nem megjeleníthető</string>  <!-- no time / time does not occur -->
```

Comment:
All of your changes look fine. 👍 I'm ready to merge this.

I noticed an existing problem at L184 (`time_none`) when testing though; "_nem megjeleníthető_" is too long (it breaks the layout). Maybe it could be "semmi" instead? Or left blank.

Reply:
Ah okay, then it would be better to write "nincs" instead.